docs/tutorials/vinyldns.md

```diff
@@ -0,0 +1,68 @@
+# Setting up ExternalDNS for VinylDNS
+
+## Create a sample deployment and service for external-dns to use
+
+Run an application and expose it via a Kubernetes Service:
+
+```console
+$ kubectl run nginx --image=nginx --replicas=1 --port=80
+$ kubectl expose deployment nginx --port=80 --target-port=80 --type=LoadBalancer
+```
+
+Annotate the Service with your desired external DNS name. Make sure to change `example.org` to your domain.
+
+```console
+$ kubectl annotate service nginx "external-dns.alpha.kubernetes.io/hostname=nginx.example.org."
+```
+
+After the service is up and running, it should get an EXTERNAL-IP. At first this may showing as `<pending>`
+
+```console
+$ kubectl get svc
+NAME         CLUSTER-IP   EXTERNAL-IP   PORT(S)        AGE
+kubernetes   10.0.0.1     <none>        443/TCP        1h
+nginx        10.0.0.115   <pending>     80:30543/TCP   10s
+```
+
+Once it's available
+
+```console
+% kubectl get svc
+NAME         CLUSTER-IP   EXTERNAL-IP   PORT(S)        AGE
+kubernetes   10.0.0.1     <none>        443/TCP        1h
+nginx        10.0.0.115   34.x.x.x      80:30543/TCP   2m
+```
+
+## Running a locally built version pointed to the above nginx service
```

Comment:
Can you also add some notes how to deploy it, see https://github.com/kubernetes-incubator/external-dns/blob/master/docs/tutorials/cloudflare.md#deploy-externaldns?

Reply:
with and without RBAC